test/transformations.jl

```diff
@@ -2,21 +2,40 @@ using StatsBase
 import StatsBase: transform, reconstruct, transform!, reconstruct!
 using Statistics
 using Test
+cuda_available = try
+    using StatsBase.CUDA
+    true
+catch
+    @warn "CUDA not available - transformations not tested on GPU"
+    false
+end
```

Comment:
These tests should probably better live in CUDA, as AFAIK CUDA isn't enabled when we run CI for StatsBase.